tests/e2e/e2e_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2022-2023, Chain4Travel AG. All rights reserved.
+//
+// This file is a derived work, based on ava-labs code whose
+// original notices appear below.
+//
+// It is distributed under the same license conditions as the
+// original code from which it is derived.
+//
+// Much love to the original authors for their work.
+// **********************************************************
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package e2e_test
+
+import (
+	"flag"
+	"testing"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+
+	"github.com/onsi/gomega"
+
+	// ensure test packages are scanned by ginkgo
+	"github.com/ava-labs/avalanchego/tests/e2e"
+	_ "github.com/ava-labs/avalanchego/tests/e2e/banff"
+	_ "github.com/ava-labs/avalanchego/tests/e2e/p"
+	_ "github.com/ava-labs/avalanchego/tests/e2e/ping"
+	_ "github.com/ava-labs/avalanchego/tests/e2e/static-handlers"
+	_ "github.com/ava-labs/avalanchego/tests/e2e/x/transfer"
+	_ "github.com/ava-labs/avalanchego/tests/e2e/x/whitelist-vtx"
+)
+
+func TestE2E(t *testing.T) {
+	gomega.RegisterFailHandler(ginkgo.Fail)
+	ginkgo.RunSpecs(t, "e2e test suites")
+}
+
+var (
+	// helpers to parse test flags
+	logLevel string
+
+	networkRunnerGRPCEp             string
+	networkRunnerCaminoNodeExecPath string
+	networkRunnerCaminoLogLevel     string
+
+	uris string
+
+	testKeysFile string
+)
+
+func init() {
+	flag.StringVar(
+		&logLevel,
+		"log-level",
+		"info",
+		"log level",
+	)
+
+	flag.StringVar(
+		&networkRunnerGRPCEp,
+		"network-runner-grpc-endpoint",
+		"",
+		"[optional] gRPC server endpoint for network-runner (only required for local network-runner tests)",
+	)
+	flag.StringVar(
+		&networkRunnerCaminoNodeExecPath,
+		"network-runner-camino-node-path",
+		"",
+		"[optional] camino-node executable path (only required for local network-runner tests)",
```

Comment:
camino-node should be replaced

Reply:
I'd suggest to keep it as it is
Its still camino node. And if we'll have different node impl in future (e.g. rust impl, as avax do), we probably can utilize the same runner